scripts/classes/deck.gd

```diff
@@ -0,0 +1,28 @@
+extends Node
```

Comment:
Since deck is not an object in the scene, but an class that just contains logic inside it, it doesn't need to inherit from Node.

So basically this line can be deleted.